api/docs/release.dox

```diff
@@ -158,7 +174,7 @@ changes:
  - Changed the 32-bit x86 stack alignment of DynamoRIO and clients built using
    DR's CMake configuration from 4 to 16 on Linux to match modern conventions.
 
-The changes between version \DR_VERSION and 7.1.0 include the following minor
+The changes between version 8.0.0 and 7.1.0 include the following minor
 compatibility changes:
 
  - Added the define #MCXT_NUM_SIMD_SLOTS that was renamed from NUM_SIMD_SLOTS.
```

Comment:
(It won't let me put a comment any further down)
If you could add to the list of non-compat changes sthg like this:

 - Fixed major synchronization bugs on ARM and AArch64.

And maybe include that in the description of the final Github Release.

Reply:
Done. What description are you referring to please?

Reply:
This one: https://github.com/DynamoRIO/dynamorio/wiki/New-Release#update-the-release-description
I.e., the one that shows up on a releases/ page (this one just says "Auto-generated periodic build."): https://github.com/DynamoRIO/dynamorio/releases/tag/cronbuild-7.91.18369

Reply:
I just updated that wiki section: normally I've been pointing at the changelog, as wsa done here: https://github.com/DynamoRIO/dynamorio/releases/tag/release_7.1.0.  I would say, do that, and also list the key highlights inlined on the releases page.  Maybe sthg like:

- Added Win10 1809 and higher support.
- Added AVX-512 support.
- Fixed major synchronization bugs on ARM and AArch64.
- Changed 32-bit Linux x86 stack alignment to 16.

Plus anything else you want to call out: those are the major ones that come to mind.

Reply:
drbbdup? :)

Reply:
Yes -- I don't see it in release.dox though: I would expect it to be there too.
Maybe also drdisas, and drcachesim function tracing enhancements.

Scanning the list I also see a typo:
```
 - Added -record_replace_retaddrd_only to drcachesim for faster function tracing
```
s/retaddrd/retaddr/

There are many small API additions that we can leave to the full changelist.

Reply:
It's there: " Added a new DR extension, namely "drbbdup", which enables different case
   instrumentation of the same basic block by duplicating code."

Reply:
Oh it is in the wrong place -- it's down under the 7.1.0 new features!  That is why I did not see it.

Reply:
Fixed!